QandA.md

```diff
@@ -59,39 +59,39 @@ Für manche fängt Non-24 im frühen Kindesalter an. Für andere beginnt es in d
 
-An dieser Stelle enthält der englische Originaltext Informationen welche sich auf die US-amerikanische Gesetzgebung zum Thema Behinderungen beziehen und wurden daher weggelassen. Andere Länder mögen ähnliche Anforderungen an Vorkehrmaßnahmen stellen.
+An dieser Stelle enthält der englische Originaltext Informationen, welche sich auf die US-amerikanische Gesetzgebung zum Thema Behinderungen beziehen und wurden daher weggelassen. Andere Länder mögen ähnliche Anforderungen an Vorkehrungen stellen.
 
 ### Welche Vorkehrungen würden helfen?
 
-Eine größere Flexibilität bei den Arbeitszeiten. Es Arbeitnehmern ermöglichen von zu Hause aus zu arbeiten, zu Zeiten an denen sie können. Kürzere Arbeitszeiten mit entsprechend weniger Entgelt erlauben, für diejenigen die das wegen chronischer Müdigkeit benötigen. Es Schüler und Studenten zu erlauben, Unterrichtsstunden, die zu nicht gewöhnlichen Zeiten stattfinden, aufgezeichnet oder online verfügbar sind, zu besuchen, oder anzuerkennen, dass sie einige Stunden versäumen werden aber trotzdem mit dem Lernstoff weitermachen können. Dafür sorgen, dass Prüfungen zu angemessenen Zeiten stattfinden. Einen leichteren Lehrplan ermöglichen, auch wenn dies bedeutet den Abschluss zu verzögern.
+Eine größere Flexibilität bei den Arbeitszeiten. Es Arbeitnehmern ermöglichen, von zu Hause aus zu arbeiten, zu Zeiten, zu denen sie können. Kürzere Arbeitszeiten mit entsprechend weniger Entgelt erlauben für diejenigen, die das wegen chronischer Müdigkeit benötigen. Es Schüler und Studenten zu erlauben, Unterrichtsstunden, die zu nicht gewöhnlichen Zeiten stattfinden, aufgezeichnet oder online verfügbar sind, zu besuchen oder anzuerkennen, dass sie einige Stunden versäumen werden aber trotzdem mit dem Lernstoff weitermachen können. Dafür sorgen, dass Prüfungen zu angemessenen Zeiten stattfinden. Einen leichteren Lehrplan ermöglichen, auch wenn dies bedeutet, den Abschluss zu verzögern.
 
 ### Beeinflusst Non-24 das Lernen in der Schule?
 
-Non-24 beeinflusst nicht die Intelligenz. Jedoch werden Lernen, Konzentration, Gedächtnis und Aufmerksamkeit von Schläfrigkeit beeinflusst. Anpassungen der Lerngewohnheiten könnten nötig sein. Dies kann am besten mit der Mithilfe vom Schulpersonal erfolgen. Es ist wichtig Kinder mit Non-24 in einem möglichst frühen Alter zu erkennen, um zu verhindern, dass sich ein Muster von Versagen entwickelt, welches mangelnde Selbstachtung begünstigt.
+Non-24 beeinflusst nicht die Intelligenz. Jedoch werden Lernen, Konzentration, Gedächtnis und Aufmerksamkeit von Schläfrigkeit beeinflusst. Anpassungen der Lerngewohnheiten könnten nötig sein. Dies kann am besten mithilfe vom Schulpersonal erfolgen. Es ist wichtig, Kinder mit Non-24 in einem möglichst frühen Alter zu erkennen, um zu verhindern, dass sich ein Muster von Misserfolgen entwickelt, welches mangelnde Selbstachtung begünstigt.
```

Comment:
- fixed grammar error
- replaced very negative and heavy term "Versagen" with the more fitting term "Misserfolge"